etc/kernel-launchers/scala/toree-launcher/build.sbt

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) Jupyter Development Team.
+ * Distributed under the terms of the Modified BSD License.
+ */
+name := "toree-launcher"
+
+version := sys.props.getOrElse("version", default = "1.0")
+
```

Comment:
This will bring the dev0 as part of the artifact version which is really a Python convention, should we use replace dev0 for SNAPSHOT which then is more towards the Java/Scala maven convention ? Changing the line above to the following should solve the problem

version := sys.props.getOrElse("version", default = "1.0").replaceAll("dev[0-9]", "SNAPSHOT")